lib/msf/ui/tip.rb

```diff
@@ -0,0 +1,43 @@
+# -*- coding: binary -*-
+# frozen_string_literal: true
+
+module Msf
+  module Ui
+    ###
+    #
+    # Module that contains some most excellent tips.
+    #
+    ###
+    module Tip
+      def self.highlight(string)
+        "%grn#{string}%clr"
+      end
+
+      ALL = [
+        "Enable verbose logging with #{highlight('set VERBOSE true')}",
+        "View useful productivity tips with the #{highlight('tip')} command, or view them all with #{highlight('tip -l')}",
+        "When in a module, use #{highlight('back')} to go back to the top level prompt",
+        "Tired of setting RHOSTS for modules? Try globally setting it with #{highlight('setg RHOSTS x.x.x.x')}",
+        "Enable HTTP request and response logging with #{highlight('set HttpTrace true')}",
+        "You can upgrade a shell to a meterpreter session on many platforms using #{highlight("sessions -u <session_id>")}",
+        "Open an interactive Ruby terminal with #{highlight('irb')}",
+        "Use the #{highlight('resource')} command to run commands from a file",
+        "To save all commands executed since start up to a file, use the #{highlight('makerc')} command",
+        "View advanced module options with #{highlight('advanced')}",
+        "You can use #{highlight('help')} to view all available commands",
+        "View a module's description using #{highlight('info')}, or view it in your browser with #{highlight('info -d')}",
+        "Writing a custom module? After editing your module, why not try the #{highlight('reload')} command",
+        "After running #{highlight('db_nmap')}, be sure to check out the result of #{highlight('hosts')} and #{highlight('services')}"
```

Comment:
```suggestion
        "After running #{highlight('db_nmap')}, be sure to check out the result of #{highlight('hosts')} and #{highlight('services')}",
        "Use the #{highlight('-x')} flag when calling Metasploit to execute commands on startup. Use #{highlight(';')} for multiple commands."
```

Reply:
I worried this might be a bit too cryptic for folk that aren't in the know - I've pointed folk to `msfconsole --help` to learn more about the start up flags as a middleground for now. I'll definitely revisit this if we can find a concise example command that teaches the concept though! :+1: